scripts/install-windows-prereqs.ps1

```diff
@@ -586,22 +611,39 @@ function Install-DCAP-Dependencies {
             if($LaunchConfiguration -eq "SGX1FLC")
             {
                 # Check if the driver is already installed and delete it
-                $output = & $devConBinaryPath find "$($drivers[${OS_VERSION}][$driver]['location'])"
+                $driverLocation = $drivers[${OS_VERSION}][$driver]['location']
+                $output = & $devConBinaryPath find $driverLocation
                 if($LASTEXITCODE) {
                     Throw "Failed searching for $driver driver"
                 }
+
                 $output | ForEach-Object {
-                    if($_.Contains($drivers[${OS_VERSION}][$driver]['description'])) {
-                        Write-Output "Removing driver $($drivers[${OS_VERSION}][$driver]['location'])"
-                        Remove-DCAPDriver -Name $drivers[${OS_VERSION}][$driver]['location']
+                    if(($_.Contains($drivers[${OS_VERSION}][$driver]['description'])) -and ($OS_VERSION -ne "WinServer2019")) {
+                        Write-Output "Removing driver $driverLocation"
+                        Remove-DCAPDriver -Name $driverLocation
                     }
                 }
-                Write-Output "Installing driver $($drivers[${OS_VERSION}][$driver]['location'])"
-                $install = & $devConBinaryPath install "$($inf.FullName)" $drivers[${OS_VERSION}][$driver]['location']
-                if($LASTEXITCODE) {
-                    Throw "Failed to install $driver driver"
+
+                if ($OS_VERSION -eq "WinServer2019")
```

Comment:
You should push this check up before the search for existing drivers, which also removes the WS2019 version check in the loop for the driver removal logic. I'd encourage you to minimize the number of branches due to the versioning logic to make updates more manageable. 

I also believe that this is not specific to WinServer2019; this code path probably should apply to Win10 and that should be tested and verified. Win10 should almost certainly not be hitting the else clause and running the Win2016 code, so I would invert the conditional check (i.e. Win2016 is the exceptional case, and Win10/WS2019 should be default/else)